### DISCUSSION TITLE
Byte ambiguous error when trying to integrate matplot in project

### DISCUSSION THREAD

**Discussion by jayanshuagrawal** [alandefreitas/matplotplusplus#271] [Q&A]
I have a project where I am trying to integrate matplot for plotting of parameters. The project also has protobuf libarary When i am trying to build my project It is throwing compilation error which says "byte: ambiguous symbol" Editor used is Visual studio 19
Can someone guide and provide solution on how to avoid this without changing the windows related files

**Comment by alandefreitas** [comment]
https://docs.microsoft.com/en-us/cpp/error-messages/compiler-errors-2/compiler-error-c2872?view=msvc-170
?
